mkinitcpio

```diff
@@ -298,15 +298,18 @@ build_uefi(){
 
     if [[ -z "$cmdline" ]]; then
         if [[ -f "/etc/kernel/cmdline" ]]; then
-            cmdline="/etc/kernel/cmdline"
+            cmdline+=("/etc/kernel/cmdline")
         elif [[ -f "/usr/lib/kernel/cmdline" ]]; then
-            cmdline="/usr/lib/kernel/cmdline"
+            cmdline+=("/usr/lib/kernel/cmdline")
         else
             warning "Note: /etc/kernel/cmdline does not exist and --cmdline is unset!"
-            cmdline="/proc/cmdline"
+            cmdline+=("/proc/cmdline")
             warning "Reusing current kernel cmdline from $cmdline"
         fi
-        msg2 "Using cmdline file: %s" "$cmdline"
+        if [[ -d "/etc/kernel/cmdline.d" ]]; then
```

Comment:
The previous `else` will match if neither `/etc/kernel/cmdline` nor `/usr/lib/kernel/cmdline` exist. This prints that `/proc/cmdline` will be use and puts if first on the list, but then the files in `cmdline.d` get read.

It would seem the values in `cmdline.d` never get effectively used.

Reply:
The else clause doesn't have anything to do with the if-section for `cmdline.d` so I'm confused why you think it won't be used? I personally use this in one of my laptops currently.